<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DienstnehmerEreignisseForm</class>
 <widget class="QDialog" name="DienstnehmerEreignisseForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dienstnehmer Ereignisse</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_newRecord">
         <property name="text">
          <string>&amp;Neu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_deleteRecord">
         <property name="text">
          <string>&amp;LÃ¶schen</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_period"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="FilterableTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilterableTableView</class>
   <extends>QTableView</extends>
   <header>lib.FilterableTableView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
